lms/envs/common.py

```diff
@@ -1223,6 +1223,8 @@
     'js/vendor/underscore-min.js',
     'js/vendor/require.js',
     'js/RequireJS-namespace-undefine.js',
+    'js/vendor/URI.min.js',
```

Comment:
Note: we should talk about this change. URI.min.js seemed to be quite small, and since we are writing everything new in Backbone, I thought it made sense to promote this to a core dependency.